src/Chirp.Razor/Pages/UserTimeline.cshtml.cs

```diff
@@ -15,7 +15,13 @@ public UserTimelineModel(ICheepService service)
 
     public ActionResult OnGet(string author)
     {
-        Cheeps = _service.GetCheepsFromAuthor(author);
+        var pageString = Request.Query["page"].ToString();
```

Comment:
Can this be done using part instead of Request.Query, and if so would that not make it easier to read?